test/bugs-reported/bug6.chpl

```diff
@@ -0,0 +1,14 @@
+use Queue;
```

Comment:
So far as I can tell, the reason this program would still fail to compile is because of the bug in the file with the Queue module in it.  Having a clean Queue module to work with would resolve the issue.

I poked around to find a good summary description of using outside modules, but the ones I found all seem to be too high level.  I have an action item to write a program for the release/examples/primers directory covering use statements, which I intend to work on shortly.